### DISCUSSION TITLE
How to use wayland environment lxqt in freebsd system?

### DISCUSSION THREAD

**Discussion by zoujiaqing** [lxqt/lxqt#2381] [Q&A]
How to use wayland environment lxqt in freebsd system?

**Comment by stefonarch** [comment]
There is no wayland-specific code/support in LXQt-components yet, but many of them can already be used, some with limitations and some not at all. I've no idea about Freebsd though,  an overview for what is at the moment possible you can find [here](https://github.com/stefonarch/LXQt-Wayland-files#wayland-implementations-for-the-lxqt-desktop).
